### DISCUSSION TITLE
What is the alternative for `setup_simulation()` in versions >= 0.6.0?

### DISCUSSION THREAD

**Discussion by darshanmeel** [atoti/atoti#502] [Q&A] [answered]
HI Guys, I would like to know what is the alternate for `cube.set_simulation()`. It seems that latest version doesn't have this but rather has `cube.create_parameter_simulation()`. This seems good as well however, I liked the set simulation and I cant seems to find that. Any help will be appreciated.
Basically I want users to create simulation scenarios by changing many parameters at the same time from the UI itself rather than doing it from the Python code.

 Also, I would like to know if we can use filter to filter the store and run some pandas calculation on filtered data and then recreate the cube.

**Comment by bloussou** [comment, marked as answer]
Hello,
The alternative is given in [0.6.0 changelog](https://docs.atoti.io/latest/releases/0.6.0.html#simulations). 

There is no longer a UI by default to modify values. You could write your own by developing an [app_extension](https://docs.atoti.io/0.6.4/lib/atoti.config.session_config.html#atoti.config.session_config.SessionConfig.app_extensions).

Regarding pandas are you talking about [`filter()`](https://docs.atoti.io/latest/lib/atoti.html#atoti.filter) ? If yes it is not modifying the table since it creates a measure (it is defined in the cube). You can load new data on a running cube using [`load_pandas()`](https://docs.atoti.io/latest/lib/atoti.table.html#atoti.table.Table.load_pandas).

**Reply by darshanmeel** [reply]
Thanks a lot. I meant to apply the filters on the store itself and not the cube. My question was if we apply the filter on the store will that automatically update the cube or do I need to recreate the cube.

**Reply by bloussou** [reply]
you cannot apply a filter on the store directly, you need to process your data before loading then load the data in the store, the cube is automatically updated when you are doing changes on the store/table, no need to do something specific for this part.
